### PR TITLE
change xml2js to import from esm.sh

### PR DESCRIPTION
**Pull request by rwblair**
one more way to skin an import

### CODE REVIEW

**Comment by rwblair** [comment]
Ok not as much fun trying to make jsr happy.

**Comment by rwblair** [comment]
files acquired with `pipx run download-esm xml2js`

File: bids-validator/deno.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "@hed/validator": "npm:hed-validator@3.15.4",
     "@ignore": "npm:ignore@5.3.2",
     "@mango/nifti": "npm:nifti-reader-js@0.6.8",
-    "@xml2js": "npm:xml2js@0.6.2"
+    "stream": "node:stream"
```
Comment (effigies) [inline bids-validator/deno.json:37]: Is this used?

**Comment by effigies** [comment]
Want to pull in 76410f7915922613e79fb9ac13e0d0fa253634d4?

File: bids-validator/deno.json (hunk 1 above)
Comment (rwblair) [reply bids-validator/deno.json:37]: Yeah xml2js dependency sax attempts to import it.

**Comment by rwblair** [comment]
`deno run -A dist/validator/bids-validator.js tests/data/bids-examples/micr_SPIM/`

This command finishes validation and prints the summary for me, but never exits. Issue doesn't exist on ds001. Moving a single ome.tif and its json into ds001 doesn't cause any problems. Seems to be stuck in denos `eventLoopTick` for node compatibility.

**Comment by rwblair** [comment]
esbuild version of validator was expecting a proper exit call, which wasn't happening for valid datasets.

**Pull request closed without merging**